### PR TITLE
Do not expose device IDs any longer

### PR DESCRIPTION
**Pull request by phsultan**
Device IDs are not exposed by `enumerateDevices` if permission to use devices has not been granted.
Reference : https://bugs.chromium.org/p/chromium/issues/detail?id=1019176
This works for Chrome, not for Firefox though.

This PR also modified `manifest.json` to:
- fix "Unrecognized manifest key applications" error
- allow extension to run on any app served by localhost

### CODE REVIEW

**Comment by phsultan** [comment]
Any many thanks for this very useful extension @fippo !

**Comment by phsultan** [comment]
Fixes #2

File: content.js [hunk 1]
```diff
@@ -71,19 +71,34 @@ var inject = '('+function() {
...
+
+                devices = devices.map((device) => {
+                    const deviceWithoutLabelAndDeviceId = {
+                        deviceId: '',
+                        kind: device.kind,
+                        label: '',
+                        groupId: device.groupId,
+                    };
+
+                    // Firefox does not empty deviceId
```
Comment (fippo) [inline content.js]:
```suggestion
                    // Firefox does not like empty deviceId
```

**Review by fippo — APPROVED** [review]

**Comment by fippo** [comment]
Thank you! Shall I squash into a single commit or do you want to squash a bit yourself?

**Comment by phsultan** [comment]
Yep, I just squashed two (six initially), it looked to me the remaining commits should be kept separate but don't know whether you'd rather keep less though. Thanks again!

**Comment by fippo** [comment]
i'm a fan of one-commit-for-thing :-)
Thank you!

**Pull request merged**